rust/actyx/node/benches/local_event_roundtrip.rs

```diff
@@ -49,7 +36,7 @@ fn round_trip(c: &mut Criterion) {
     }
     c.bench_function("id", |b| {
         b.to_async(&rt).iter_batched(
-            || (data.clone(), mk_http_client()),
+            || (data.clone(), ActyxClient::new(AxClientOpts::default())),
```

Comment:
How about `Ax::client(AxOpts::default())`?

Reply:
I wasn't changing the names for the client yet. I can do it though